documentation/docs/css/rating.css

```diff
@@ -0,0 +1,43 @@
+.rating-widget:hover,
+#feedback-form:hover {
+  box-shadow:
+    0px 1px 10px 0px rgba(0, 0, 0, 0.12),
+    0px 4px 5px 0px rgba(0, 0, 0, 0.14),
+    0px 2px 4px -1px rgba(0, 0, 0, 0.20);
+}
+
+/* Title color */
+.rating-widget .rating-title {
+  color: var(--md-typeset-a-color, #0E5FB5);
+}
+
+/* Dark mode title */
+@media (prefers-color-scheme: dark) {
+  .rating-widget .rating-title {
+    color: var(--md-typeset-a-color, #93C7FF) !important;
```

Comment:
Given that `!important` makes it hard to maintain the code, can we get rid of it by providing higher specificity selectors?

Reply:
Well, I'm still seeing many of those in https://github.com/percona/pmm/pull/4765/files#diff-8649e1fc5c9c5bbf23d0ccdacc4a2265f2f5f63c7fda360e4255412c4c9b8fd7R163-R190